dist/info/duckstation_libretro.info

```diff
@@ -1,10 +1,10 @@
 # Software Information
 display_name = "Sony - PlayStation (DuckStation)"
 authors = "stenzek"
-supported_extensions = "cue|bin|img|chd|pbp|m3u"
+supported_extensions = "exe|psexe|cue|bin|img|iso|chd|pbp|ecm|mds|psf|m3u"
 corename = "DuckStation"
 categories = "Emulator"
-license = "GPLv3"
+license = "Non-commercial"
```

Comment:
@stenzek Hi, Could you clarify license? Something changed, it's under multiple license or no longer GPL-3.0?

Regards,
Bartłomiej